dist/main.js

```diff
@@ -0,0 +1,62 @@
+const weatherApp = new WeatherAPP
+const renderer = new Renderer
+const background = document.querySelector('#background')
+
+if (navigator.geolocation) {
+  navigator.geolocation.getCurrentPosition((position) =>{
+     weatherApp.coords = { 
+         lat: position.coords.latitude,
+         lon: position.coords.longitude
+     }
+    
+     weatherApp.getCityByCoord().then(function (city) {
+       if (city) {
+         renderer.renderSearchData(city)
+       }else{
+         weatherApp.searchCity = weatherApp.data[0]
+         renderer.renderSearchData(weatherApp.searchCity)
+       }
+     })
+ })
+}
+
+weatherApp.loadFavCities().then(function (cities) {
+  renderer.renderFavCities(cities)
+})
+
+$('#searchBtn').on('click', async function () {
```

Comment:
might want to remove the functions from the event listener. I find it easier to read. Especially since you have so many event listeners.
`searchCity = () => {}`

`$('#searchBtn').on('click', searchCity)`